2024-03-21T07:52:12 {"cpu": 2.6572265625, "cpu_total": 8, "cpu_usage": 0.3321533203125, "memory": 1885.0, "memory_total": 8865.0, "memory_usage": 0.212633953750705, "pipeline_run_id": 6165, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Pipeline etl_pipeline for run 6165 in schedule 3 is alive.", "timestamp": 1711007532.065139, "uuid": "b3037b7f75f04d6f8dc3ed259daae996"}
2024-03-21T07:52:12 {"block_run_id": 18490, "block_uuid": "api_load_data", "pipeline_run_id": 6165, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Execute PipelineRun 6165, BlockRun 18490: pipeline etl_pipeline block api_load_data", "timestamp": 1711007532.15055, "uuid": "1723d43b6df042ebb574cb590cea3ca1"}
2024-03-21T07:52:12 {"cpu": 2.6572265625, "cpu_total": 8, "cpu_usage": 0.3321533203125, "memory": 1934.0, "memory_total": 8865.0, "memory_usage": 0.21816130851663845, "pipeline_run_id": 6165, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Pipeline etl_pipeline for run 6165 in schedule 3 is alive.", "timestamp": 1711007532.194677, "uuid": "76d1feb592594d40bd8ec3dd05ef17e5"}
2024-03-21T07:52:12 {"block_run_id": 18490, "block_uuid": "api_load_data", "pipeline_run_id": 6165, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "BlockRun 18490 (block_uuid: api_load_data) completes.", "timestamp": 1711007532.833217, "uuid": "427e6259eef94ff1b420d3349d67b443"}
2024-03-21T07:52:21 {"cpu": 2.66650390625, "cpu_total": 8, "cpu_usage": 0.33331298828125, "memory": 1891.0, "memory_total": 8865.0, "memory_usage": 0.21331077270163565, "pipeline_run_id": 6165, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Pipeline etl_pipeline for run 6165 in schedule 3 is alive.", "timestamp": 1711007541.367434, "uuid": "ad3c1d065a674937a754f59c964db817"}
2024-03-21T07:52:21 {"block_run_id": 18491, "block_uuid": "transform_data", "pipeline_run_id": 6165, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Execute PipelineRun 6165, BlockRun 18491: pipeline etl_pipeline block transform_data", "timestamp": 1711007541.461165, "uuid": "257fce8a1071469b861178dcc4c830d5"}
2024-03-21T07:52:21 {"block_run_id": 18491, "block_uuid": "transform_data", "pipeline_run_id": 6165, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "BlockRun 18491 (block_uuid: transform_data) completes.", "timestamp": 1711007541.54763, "uuid": "bc11e240e2a54eadb6a0d0f6d346cda7"}
2024-03-21T07:52:31 {"cpu": 2.68115234375, "cpu_total": 8, "cpu_usage": 0.33514404296875, "memory": 1885.0, "memory_total": 8865.0, "memory_usage": 0.212633953750705, "pipeline_run_id": 6165, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Pipeline etl_pipeline for run 6165 in schedule 3 is alive.", "timestamp": 1711007551.589778, "uuid": "3dacc45ea10243ae9664029c673812e9"}
2024-03-21T07:52:31 {"block_run_id": 18492, "block_uuid": "export_to_postgres", "pipeline_run_id": 6165, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Execute PipelineRun 6165, BlockRun 18492: pipeline etl_pipeline block export_to_postgres", "timestamp": 1711007551.691617, "uuid": "025cb2101e1d4c2cba5caffcf96c6f62"}
2024-03-21T07:52:31 {"block_run_id": 18492, "block_uuid": "export_to_postgres", "pipeline_run_id": 6165, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "EXCEPTION", "message": "BlockRun 18492 (block_uuid: export_to_postgres) failed.", "timestamp": 1711007551.774627, "uuid": "940a6db7159e4db3843ca2128116cfc0", "error": ["Traceback (most recent call last):\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py\", line 613, in execute\n    result = __execute_with_retry()\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/shared/retry.py\", line 54, in retry_func\n    raise e\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/shared/retry.py\", line 38, in retry_func\n    return func(*args, **kwargs)\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py\", line 588, in __execute_with_retry\n    return self._execute(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py\", line 1077, in _execute\n    result = self.block.execute_sync(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1314, in execute_sync\n    raise err\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1223, in execute_sync\n    output = self.execute_block(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1529, in execute_block\n    outputs = self._execute_block(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1685, in _execute_block\n    outputs = self.execute_block_function(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1724, in execute_block_function\n    output = block_function_updated(*input_vars, **global_vars)\n  File \"<string>\", line 25, in export_data_to_postgres\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/io/sql.py\", line 279, in export\n    df = clean_df_for_export(df, self.clean, dtypes)\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/io/export_utils.py\", line 90, in clean_df_for_export\n    copy_df = df.copy()\nAttributeError: 'int' object has no attribute 'copy'\n"], "error_stack": [["  File \"/usr/local/bin/mage\", line 8, in <module>\n    sys.exit(app())\n", "  File \"/usr/local/lib/python3.10/site-packages/typer/main.py\", line 311, in __call__\n    return get_command(self)(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/click/core.py\", line 1130, in __call__\n    return self.main(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/typer/core.py\", line 778, in main\n    return _main(\n", "  File \"/usr/local/lib/python3.10/site-packages/typer/core.py\", line 216, in _main\n    rv = self.invoke(ctx)\n", "  File \"/usr/local/lib/python3.10/site-packages/click/core.py\", line 1657, in invoke\n    return _process_result(sub_ctx.command.invoke(sub_ctx))\n", "  File \"/usr/local/lib/python3.10/site-packages/click/core.py\", line 1404, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n", "  File \"/usr/local/lib/python3.10/site-packages/click/core.py\", line 760, in invoke\n    return __callback(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/typer/main.py\", line 683, in wrapper\n    return callback(**use_params)  # type: ignore\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/cli/main.py\", line 163, in start\n    start_server(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/server/server.py\", line 743, in start_server\n    scheduler_manager.start_scheduler()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/server/scheduler_manager.py\", line 87, in start_scheduler\n    proc.start()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 121, in start\n    self._popen = self._Popen(self)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 224, in _Popen\n    return _default_context.get_context().Process._Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 281, in _Popen\n    return Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 19, in __init__\n    self._launch(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 71, in _launch\n    code = process_obj._bootstrap(parent_sentinel=child_r)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 314, in _bootstrap\n    self.run()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 108, in run\n    self._target(*self._args, **self._kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/process.py\", line 15, in start_session_and_run\n    results = target(*args)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/server/scheduler_manager.py\", line 50, in run_scheduler\n    LoopTimeTrigger().start()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/triggers/loop_time_trigger.py\", line 14, in start\n    self.run()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/triggers/time_trigger.py\", line 11, in run\n    schedule_all()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 1643, in schedule_all\n    PipelineScheduler(r).schedule()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/__init__.py\", line 157, in func_with_rollback\n    return func(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 318, in schedule\n    self.__schedule_blocks(block_runs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 602, in __schedule_blocks\n    job_manager.add_job(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/job_manager.py\", line 28, in add_job\n    self.queue.enqueue(job_id, target, *args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/queue/process_queue.py\", line 108, in enqueue\n    self.start_worker_pool()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/queue/process_queue.py\", line 183, in start_worker_pool\n    self.worker_pool_proc.start()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 121, in start\n    self._popen = self._Popen(self)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 224, in _Popen\n    return _default_context.get_context().Process._Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 281, in _Popen\n    return Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 19, in __init__\n    self._launch(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 71, in _launch\n    code = process_obj._bootstrap(parent_sentinel=child_r)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 314, in _bootstrap\n    self.run()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 108, in run\n    self._target(*self._args, **self._kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/queue/process_queue.py\", line 288, in poll_job_and_execute\n    worker.start()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 121, in start\n    self._popen = self._Popen(self)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 224, in _Popen\n    return _default_context.get_context().Process._Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 281, in _Popen\n    return Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 19, in __init__\n    self._launch(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 71, in _launch\n    code = process_obj._bootstrap(parent_sentinel=child_r)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 314, in _bootstrap\n    self.run()\n", "  File \"/usr/local/lib/python3.10/site-packages/newrelic/api/background_task.py\", line 117, in wrapper\n    return wrapped(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/queue/process_queue.py\", line 253, in run\n    start_session_and_run(args[1], *args[2], **args[3])\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/process.py\", line 15, in start_session_and_run\n    results = target(*args)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 1152, in run_block\n    return ExecutorFactory.get_block_executor(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py\", line 641, in execute\n    on_failure(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/__init__.py\", line 157, in func_with_rollback\n    return func(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 433, in on_block_failure\n    self.logger.exception(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/logging/logger.py\", line 30, in exception\n    self.__send_message('exception', message, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/logging/logger.py\", line 65, in __send_message\n    data['error_stack'] = traceback.format_stack(),\n"]], "error_stacktrace": ["'int' object has no attribute 'copy'"]}
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py", line 613, in execute
    result = __execute_with_retry()
  File "/usr/local/lib/python3.10/site-packages/mage_ai/shared/retry.py", line 54, in retry_func
    raise e
  File "/usr/local/lib/python3.10/site-packages/mage_ai/shared/retry.py", line 38, in retry_func
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py", line 588, in __execute_with_retry
    return self._execute(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py", line 1077, in _execute
    result = self.block.execute_sync(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1314, in execute_sync
    raise err
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1223, in execute_sync
    output = self.execute_block(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1529, in execute_block
    outputs = self._execute_block(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1685, in _execute_block
    outputs = self.execute_block_function(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1724, in execute_block_function
    output = block_function_updated(*input_vars, **global_vars)
  File "<string>", line 25, in export_data_to_postgres
  File "/usr/local/lib/python3.10/site-packages/mage_ai/io/sql.py", line 279, in export
    df = clean_df_for_export(df, self.clean, dtypes)
  File "/usr/local/lib/python3.10/site-packages/mage_ai/io/export_utils.py", line 90, in clean_df_for_export
    copy_df = df.copy()
AttributeError: 'int' object has no attribute 'copy'
2024-03-21T07:52:41 {"cpu": 2.67333984375, "cpu_total": 8, "cpu_usage": 0.33416748046875, "memory": 1887.0, "memory_total": 8865.0, "memory_usage": 0.2128595600676819, "pipeline_run_id": 6165, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Pipeline etl_pipeline for run 6165 in schedule 3 is alive.", "timestamp": 1711007561.829318, "uuid": "a2cdbe87e5ee4b7dbff5686ef520df61"}
2024-03-21T07:58:01 {"cpu": 2.6064453125, "cpu_total": 8, "cpu_usage": 0.3258056640625, "memory": 1903.0, "memory_total": 8865.0, "memory_usage": 0.2146644106034969, "pipeline_run_id": 6172, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Pipeline etl_pipeline for run 6172 in schedule 3 is alive.", "timestamp": 1711007881.146285, "uuid": "a36bb78386f140c484a8da984767d15e"}
2024-03-21T07:58:01 {"block_run_id": 18511, "block_uuid": "api_load_data", "pipeline_run_id": 6172, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Execute PipelineRun 6172, BlockRun 18511: pipeline etl_pipeline block api_load_data", "timestamp": 1711007881.251426, "uuid": "3c2e04766594460ea61b4a65a7c1ebfa"}
2024-03-21T07:58:01 {"cpu": 2.6064453125, "cpu_total": 8, "cpu_usage": 0.3258056640625, "memory": 2030.0, "memory_total": 8865.0, "memory_usage": 0.22899041173152848, "pipeline_run_id": 6172, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Pipeline etl_pipeline for run 6172 in schedule 3 is alive.", "timestamp": 1711007881.390066, "uuid": "2c04c22eeff24324ac846f41fa7c1e11"}
2024-03-21T07:58:01 {"block_run_id": 18511, "block_uuid": "api_load_data", "pipeline_run_id": 6172, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "BlockRun 18511 (block_uuid: api_load_data) completes.", "timestamp": 1711007881.58547, "uuid": "7f147828113a43fe94a19f04f61114b9"}
2024-03-21T07:58:11 {"cpu": 2.61083984375, "cpu_total": 8, "cpu_usage": 0.32635498046875, "memory": 1932.0, "memory_total": 8865.0, "memory_usage": 0.2179357021996616, "pipeline_run_id": 6172, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Pipeline etl_pipeline for run 6172 in schedule 3 is alive.", "timestamp": 1711007891.553711, "uuid": "f03e843de3f94a9f8154e1e69f8e19f7"}
2024-03-21T07:58:11 {"block_run_id": 18512, "block_uuid": "transform_data", "pipeline_run_id": 6172, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Execute PipelineRun 6172, BlockRun 18512: pipeline etl_pipeline block transform_data", "timestamp": 1711007891.642179, "uuid": "950cc64a4fe34fe7a972bbc25860b1bd"}
2024-03-21T07:58:11 {"block_run_id": 18512, "block_uuid": "transform_data", "pipeline_run_id": 6172, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "BlockRun 18512 (block_uuid: transform_data) completes.", "timestamp": 1711007891.751654, "uuid": "436f16039e4947c1a1f984c840ee67f2"}
2024-03-21T07:58:21 {"cpu": 2.615234375, "cpu_total": 8, "cpu_usage": 0.326904296875, "memory": 1926.0, "memory_total": 8865.0, "memory_usage": 0.21725888324873097, "pipeline_run_id": 6172, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Pipeline etl_pipeline for run 6172 in schedule 3 is alive.", "timestamp": 1711007901.780086, "uuid": "95592e4d52304d12ac6f07c7f76b9d22"}
2024-03-21T07:58:21 {"block_run_id": 18513, "block_uuid": "export_to_postgres", "pipeline_run_id": 6172, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Execute PipelineRun 6172, BlockRun 18513: pipeline etl_pipeline block export_to_postgres", "timestamp": 1711007901.86727, "uuid": "60962510de7643c2b6d7e72be53360ea"}
2024-03-21T07:58:22 {"block_run_id": 18513, "block_uuid": "export_to_postgres", "pipeline_run_id": 6172, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "BlockRun 18513 (block_uuid: export_to_postgres) completes.", "timestamp": 1711007902.012719, "uuid": "110e1a1adc3a42cd896ccde33e8eb703"}
2024-03-21T07:58:31 {"cpu": 2.6142578125, "cpu_total": 8, "cpu_usage": 0.3267822265625, "memory": 1925.0, "memory_total": 8865.0, "memory_usage": 0.21714608009024253, "pipeline_run_id": 6172, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Pipeline etl_pipeline for run 6172 in schedule 3 is alive.", "timestamp": 1711007911.982477, "uuid": "360b50c816ed4aea932265609a072456"}
2024-03-21T08:00:33 {"cpu": 2.62158203125, "cpu_total": 8, "cpu_usage": 0.32769775390625, "memory": 1951.0, "memory_total": 8865.0, "memory_usage": 0.22007896221094192, "pipeline_run_id": 6179, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Pipeline etl_pipeline for run 6179 in schedule 3 is alive.", "timestamp": 1711008033.053542, "uuid": "b1c9d37479ce4bb6a3ce4cf0dddf70e3"}
2024-03-21T08:00:33 {"block_run_id": 18532, "block_uuid": "api_load_data", "pipeline_run_id": 6179, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Execute PipelineRun 6179, BlockRun 18532: pipeline etl_pipeline block api_load_data", "timestamp": 1711008033.156101, "uuid": "cb8fd9216c0745c88c0d8ff9eba8ae85"}
2024-03-21T08:00:33 {"cpu": 2.62158203125, "cpu_total": 8, "cpu_usage": 0.32769775390625, "memory": 1988.0, "memory_total": 8865.0, "memory_usage": 0.2242526790750141, "pipeline_run_id": 6179, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Pipeline etl_pipeline for run 6179 in schedule 3 is alive.", "timestamp": 1711008033.20151, "uuid": "bd3bc20fbf954144914ad8f093229676"}
2024-03-21T08:00:34 {"block_run_id": 18532, "block_uuid": "api_load_data", "pipeline_run_id": 6179, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "BlockRun 18532 (block_uuid: api_load_data) completes.", "timestamp": 1711008034.03095, "uuid": "64c6f03e6eb84d58a5c6a8f76079fed9"}
2024-03-21T08:00:42 {"cpu": 2.6240234375, "cpu_total": 8, "cpu_usage": 0.3280029296875, "memory": 1951.0, "memory_total": 8865.0, "memory_usage": 0.22007896221094192, "pipeline_run_id": 6179, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Pipeline etl_pipeline for run 6179 in schedule 3 is alive.", "timestamp": 1711008042.397445, "uuid": "362f4cde80c740518af353745245628a"}
2024-03-21T08:00:42 {"block_run_id": 18533, "block_uuid": "transform_data", "pipeline_run_id": 6179, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "Execute PipelineRun 6179, BlockRun 18533: pipeline etl_pipeline block transform_data", "timestamp": 1711008042.50082, "uuid": "ba9cb3eff38248919b3dafd1797a0b27"}
2024-03-21T08:00:42 {"block_run_id": 18533, "block_uuid": "transform_data", "pipeline_run_id": 6179, "pipeline_schedule_id": 3, "pipeline_uuid": "etl_pipeline", "hostname": "3353dc602115", "level": "INFO", "message": "BlockRun 18533 (block_uuid: transform_data) completes.", "timestamp": 1711008042.636623, "uuid": "27b036156cf04035957375ff66c15ff7"}
